matlab/lib/+Ice/Communicator.m

```diff
@@ -25,14 +25,14 @@
     % Copyright (c) ZeroC, Inc.
 
     methods
-        function [obj, remArgs] = Communicator(args, options)
+        function obj = Communicator(args, options)
             %COMMUNICATOR Constructs a new communicator.
             %
             %   Examples
             %     communicator = Ice.Communicator();
-            %     [communicator, remArgs] = Ice.Communicator(args);
+            %     communicator = Ice.Communicator(args);
             %     communicator = Ice.Communicator(Properties = props, SliceLoader = sliceLoader);
-            %     [communicator, remArgs] = Ice.Communicator(args, SliceLoader = sliceLoader);
+            %     communicator = Ice.Communicator(args, Properties = props, SliceLoader = sliceLoader);
```

Comment:
I kept args + Properties because I could not find a way to use an arguments block (see below) and disallow this combination. I think that's fine.